CHANGELOG.md

```diff
@@ -1,25 +1,30 @@
-## 1.0.2
+# Changelog
+All notable changes to this project will be documented in this file.
 
-### FIXED
-
-- Fixed deprecation notice for `google-beta` provider in `core_project_factory` module.
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 1.0.1
+## [Unreleased]
+### ADDED
+- Preconditions script checks billing account format.
```

Comment:
@morgante just added PR links/numbers. This doesn't seem to be part of the spec. Do you think it would make sense to add a note after the "...The format is based on..." line at the top?